---
title: "References"
author: "Daniel Gardner"
date: "2025-01-27"
output: html_document
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

# References

The main paper our research is based off, and which describes the pipeline as shown in the three bubbles is:

-   *[B*é*al et al., 2019] B*é*al, J., Montagud, A., Traynard, P., Barillot, E., and Calzone, L. (2019). Personalization of logical models with multi-omics data allows clinical stratifi-cation of patients. Frontiers in Physiology, 9.*

The example Boolean Network is taken from the paper:

-   *[Ruscone et al., 2023] Ruscone, M., Montagud, A., Chavrier, P., Destaing, O., Bonnet, I., Zinovyev, A., Barillot, E., No¨el, V., and Calzone, L. (2023). Multiscale model of the different modes of cancer cell invasion. Bioinformatics, 39(6):btad374.*

The example slides of lung cancer samples are taken from:

-   *[Quiros et al., 2024] Quiros, A., Coudray, N., Yeaton, A., Yang, X., Liu, B., Le, H., Chiriboga, L., Karimkhan, A., Narula, N., Moore, D., Park, C., Pass, H., Moreira, A., Quesne, J., Tsirigos, A., and Yuan, K. (2024). Mapping the landscape of histomorphological cancer phenotypes using self-supervised learning on unannotated pathology slides. NatureCommunications, 15.*

We also reference two databases, KEGG(<https://www.genome.jp/kegg/>):

-   *[Kanehisa and Goto, 2000] Kanehisa, M. and Goto, S. (2000). Kegg: Kyoto encyclopedia of genes and genomes. Nucleic Acids Research, 28(1):27--30.*

And Reactome (<https://reactome.org/>):

-   *[Milacic et al., 2023] Milacic, M., Beavers, D., Conley, P., Gong, C., Gillespie, M., Griss, J., Haw, R., Jassal, B., Matthews, L., May, B., Petryszak, R., Ragueneau, E., Rothfels, K., Sevilla, C., Shamovsky, V., Stephan, R., Tiwari, K., Varusai, T., Weiser, J., Wright, A., Wu, G., Stein, L., Hermjakob, H., and D'Eustachio, P. (2023). The reactome pathway knowledgebase 2024. Nucleic Acids Research, 52(D1):D672--D678.*

As well as the ROMA method:

-   *[Martignetti et al., 2016] Martignetti, L., Calzone, L., Bonnet, E., Barillot, E., and Zinovyev, A. (2016). Roma: Representation and quantification of module activity from target expression data. Frontiers in Genetics, 7.*

## Further Reading

For a general introduction to Boolean networks, we would recommend:

-   *[Abou-Jaoudé et al., 2016] Abou-Jaoudé, W., Traynard, P., Monteiro, P. T., Saez-Rodrıguez, J., Helikar, T., Thieffry, D., and Chaouiya, C. (2016). Logical modeling and dynamical analysis of cellular networks. Frontiers in Genetics, 7.*

-   *[Stoll et al., 2012] Stoll, G., Viara, E., Barillot, E., and Calzone, L. (2012). Continuous time boolean modeling for biological signaling: application of gillespie algorithm.*

For a guide to applying Boolean networks to biological contexts, please see:

-   *[Cohen et al., 2015] Cohen, D. P. A., Martignetti, L., Robine, S., Barillot, E., Zinovyev, A., and Calzone, L. (2015). Mathematical modelling of molecular pathways enabling tumour cell invasion and migration. PLOS Computational Biology, 11.*

And some papers we are currently using as inspiration for training models to data are:

-   *[Saez-Rodriguez et al., 2009] Saez-Rodriguez, J., Alexopoulos, L. G., Epperlein, J., Samaga, R., Lauffenburger, D. A., Klamt, S., and Sorger, P. K. (2009). Discrete logic modelling as a means to link protein signalling networks with functional analysis of mammalian signal transduction. Molecular Systems Biology, 5(1):331.*

-   *[Ostrowski et al., 2016] Ostrowski, M., Pauleve, L., Schaub, T., Siegel, A., and Guziolowski, C. (2016). Boolean network identification from perturbation time series data combining dynamics abstraction and logic programming. Biosystems, 149:139--153. Selected papers from the Computational Methods in Systems Biology 2015 conference.*

-   *[Razzaq et al., 2018] Razzaq, M., Paulev´e, L., Siegel, A., Saez-Rodriguez, J., Bourdon, J., and Guziolowski, C. (2018). Computational discovery of dynamic cell line specific boolean networks from multiplex time-course data. PLOS Computational Biology, 14(10):1--23.*
